INSTALL.md

```diff
@@ -0,0 +1,268 @@
+# Installing google-cloud-cpp
+
+By default google-cloud-cpp libraries download and compile all their
+dependencies. This makes it easier for users to "take the library for a spin",
```

Comment:
Specifying which tools it supports downloading and compiling all the necessary dependencies would help here.

Reply:
You mean which dependencies it downloads? I added a link, let me know if that is not what you had in mind.

Reply:
I meant to delete this comment. Please ignore.